app/src/main/java/com/example/tkey_android/FirstFragment.java

```diff
@@ -299,95 +341,176 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
                             Exception e = ((com.web3auth.tkey.ThresholdKey.Common.Result.Error<Void>) result).exception;
                             Snackbar snackbar = Snackbar.make(view1, "A problem occurred: " + e.getMessage(), Snackbar.LENGTH_LONG);
```

Comment:
Please make these consistent with the `renderError(e);` you're using
```suggestion
                            Snackbar snackbar = Snackbar.make(view1, "A problem occurred: " + e.getMessage(), Snackbar.LENGTH_LONG);
```